### DISCUSSION TITLE
I don't see the cheat menu when I open up Prodigy.

### DISCUSSION THREAD

**Discussion by M708591** [Prodigy-Hacking/ProdigyMathGameHacking#955] [(B) Bug Discussion]
When I got into Prodigy I was going to use the cheat menu so I could win a battle, but I didn't see it. I thought refreshing my screen would fix it, but nothing happened. After that, I deleted the extension and redownloaded it, still nothing happened.

**Comment by ghost** [comment]
you allso have to drag it into extinsions ust downloading it wont work
